x-pack/test/cases_api_integration/common/plugins/cases/server/routes.ts

```diff
@@ -110,6 +110,34 @@ export const registerRoutes = (core: CoreSetup<FixtureStartDeps>, logger: Logger
     }
   );
 
+  router.get(
```

Comment:
Do you plan to remove this later on? It might be one of those cases where adding a comment about why this exists might be useful for the future response ops team

Reply:
Good point. I will add a comment for reference. 
I will create an issue to remove case client methods like these from test framework.

Reply:
Done. Issue: https://github.com/elastic/kibana/issues/208188